components/author/index.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
I don't love this. What was the reason to disable these?

Reply:
Ohhh gosh no. That rule should never be disabled. 😱 What was I thinking  😂 Will review 👍